jobs/config.json

```diff
@@ -9960,7 +9960,7 @@
   },
   "pull-kubernetes-e2e-gce": {
     "args": [
-      "--build=bazel",
+      "--build",
```

Comment:
I don't love this, but it does let us run the job on jenkins while overriding the args in Prow which we have better support for now.